airbyte-webapp/src/services/job/JobService.tsx

```diff
@@ -25,9 +25,11 @@ function useGetJobService() {
 
 export const useListJobs = (listParams: JobListRequestBody) => {
   const service = useGetJobService();
-  return useSuspenseQuery(jobsKeys.list(listParams.configId), () => service.list(listParams), {
+  const result = useQuery(jobsKeys.list(listParams.configId, listParams.pagination), () => service.list(listParams), {
     refetchInterval: 2500, // every 2,5 seconds,
-  }).jobs;
+    keepPreviousData: true,
```

Comment:
We'd now want to pass the `suspense: true` here, that the `useSuspenseQuery` usually set. This will cause React to not render the component (beyond the point of this method call) as long as the initial data is still loaded, and instead show the `fallback` element of the `React.Suspense` next up in the React tree.

Reply:
Done